rwengine/src/script/modules/GTA3ModuleImpl.inl

```diff
@@ -28,8 +31,13 @@ void opcode_0001(const ScriptArguments& args, const ScriptInt time) {
 */
 void opcode_0002(const ScriptArguments& args, const ScriptLabel arg1) {
 	RW_UNIMPLEMENTED_OPCODE(0x0002);
-	RW_UNUSED(arg1);
-	RW_UNUSED(args);
+	auto thread = args.getThread();
+	if (arg1 < 0) {
```

Comment:
Yes.
On 5 Jul 2016 12:54 am, "Jannik Vogel" <notifications@github.com> wrote:

> In rwengine/src/script/modules/GTA3ModuleImpl.inl
> <https://github.com/danhedron/openrw/pull/1#discussion_r69496106>:
>
> > @@ -28,8 +31,13 @@ void opcode_0001(const ScriptArguments& args, const ScriptInt time) {
> >  */
> >  void opcode_0002(const ScriptArguments& args, const ScriptLabel arg1) {
> >  	RW_UNIMPLEMENTED_OPCODE(0x0002);
> > -	RW_UNUSED(arg1);
> > -	RW_UNUSED(args);
> > +	auto thread = args.getThread();
> > +	if (arg1 < 0) {
>
> shouldn't this be handled by the ScriptLabel parser?
> [or a function of the ScriptLabel type]
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/danhedron/openrw/pull/1/files/fd47d4ce6c1ab595dcf8b9223baf29344f02044f..7822cf953b309a4d6fb24e278ca0888ca2108e97#r69496106>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAZho99DbOlBrDLSRHPezSUSLm48Tm0pks5qSZ04gaJpZM4IuZLC>
> .
>